### PR TITLE
fix(TBD-9218): fix ClassLoader issues

### PR DESCRIPTION
**Pull request by sponomarova**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

see TBD-9218

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.designer.components.bigdata/components/tSqoopMerge/tSqoopMerge_javaapi.javajet [hunk 1]
```diff
@@ -157,12 +157,14 @@
...
 %>
-
-    java.net.URLClassLoader sysloader_<%=cid%> = (java.net.URLClassLoader) ClassLoader.getSystemClassLoader();
-    java.lang.reflect.Method method_<%=cid%> = java.net.URLClassLoader.class.getDeclaredMethod("addURL", new Class[] { java.net.URL.class});
-    method_<%=cid%>.setAccessible(true);
+	java.util.List<java.net.URL> libURL_<%=cid%> = new java.util.ArrayList<>();
+	
+    java.net.URL[] libURLArray_<%=cid%> = libURL_<%=cid%>.toArray(new java.net.URL[] {});
+	ClassLoader threadClassLoader_<%=cid%> = Thread.currentThread().getContextClassLoader();
+	java.net.URLClassLoader newthreadClassLoader_<%=cid%> = new java.net.URLClassLoader(libURLArray_<%=cid%>, threadClassLoader_<%=cid%>);
```
Comment (lbourgeois) [inline main/plugins/org.talend.designer.components.bigdata/components/tSqoopMerge/tSqoopMerge_javaapi.javajet]: Seems libURLArray_<%=cid%> is empty here + jarFileName_<%=cid%> from original code is never used

Comment (sponomarova) [reply main/plugins/org.talend.designer.components.bigdata/components/tSqoopMerge/tSqoopMerge_javaapi.javajet]: PR is updated.

File: main/plugins/org.talend.designer.components.bigdata/components/tSqoopMerge/tSqoopMerge_javaapi.javajet [hunk 2]
```diff
@@ -157,12 +157,13 @@
 <%
     }
 %>
-
-    java.net.URLClassLoader sysloader_<%=cid%> = (java.net.URLClassLoader) ClassLoader.getSystemClassLoader();
-    java.lang.reflect.Method method_<%=cid%> = java.net.URLClassLoader.class.getDeclaredMethod("addURL", new Class[] { java.net.URL.class});
-    method_<%=cid%>.setAccessible(true);
+	java.util.List<java.net.URL> libURL_<%=cid%> = new java.util.ArrayList<>();
```
Comment (lbourgeois) [inline main/plugins/org.talend.designer.components.bigdata/components/tSqoopMerge/tSqoopMerge_javaapi.javajet]: libURL_<%=cid%> variable seems never used

Comment (sponomarova) [reply main/plugins/org.talend.designer.components.bigdata/components/tSqoopMerge/tSqoopMerge_javaapi.javajet]: Agree. Will remove it.

Comment (sponomarova) [reply main/plugins/org.talend.designer.components.bigdata/components/tSqoopMerge/tSqoopMerge_javaapi.javajet]: Already removed.

**Pull request merged**